channel/channel_test.go

```diff
@@ -76,7 +73,7 @@ var _ = Describe("Channels", func() {
 					if inOrder {
 						Expect(data).To(Equal(iter))
 					}
-					if rand.Int()%1000 == 0 {
+					if rand.Int()%100 == 0 {
```

Comment:
We probably don't want this much logging outside of debugging.
```suggestion
					if rand.Int()%1000 == 0 {
```